daffodil-core/src/main/scala/org/apache/daffodil/grammar/primitives/PrimitivesElementKinds.scala

```diff
@@ -113,22 +113,21 @@ case class SequenceCombinator(sq: SequenceTermBase, rawTerms: Seq[Gram])
   extends Terminal(sq, true) {
 
   override lazy val isEmpty = {
-    val rt = rawTerms
-    val frt = rt.filterNot { _.isEmpty }
-    val res = frt.isEmpty
+    val ft = terms.filterNot { _.isEmpty }
+    val res = ft.isEmpty
     res
```

Comment:
This looks odd to me. terms is ``rawTerms.filterNot { _.isEmpty }`` and then you do ``terms.filterNot { _.isEmpty }``. Seems like that second filterNot will do nothing, so can this just be ``val res = terms.isEmpty``?